modules/configuring/proc-adding-custom-configuration-files-by-using-helm.adoc

```diff
@@ -0,0 +1,31 @@
+[id='proc-add-custom-app-file-openshift-helm_{context}']
+= Adding custom {product} configuration files by using the Helm chart
```

Comment:
Similar to the operator title above.
I think that this would describe it better:

```suggestion
= Adding custom {product} configuration files when using the Helm chart
```